src/libsystemd/sd-journal/journal-send.c

```diff
@@ -318,7 +332,7 @@ _public_ int sd_journal_sendv(const struct iovec *iov, int n) {
         if (errno == ENOENT)
                 return 0;
 
-        if (!IN_SET(errno, EMSGSIZE, ENOBUFS))
+        if (!IN_SET(errno, EMSGSIZE, ENOBUFS, EAGAIN))
```

Comment:
Is it not necessary to handle `EAGAIN` from `send_one_fd_sa()` below?

If `EAGAIN` from the two `sendmsg()` are ignored, then it is not necessary to handle the error in `submit_coredump()`.

Reply:
We handle it here so that we go to the fallback code which uses an fd which might fit in the socket send buffer. If that fails, I think we want to fail with EAGAIN to inform the caller that we were unable to send the log to the journal. I don't think we should swallow the error in that case.